packit_service/worker/whitelist.py

```diff
@@ -214,15 +213,23 @@ def check_and_report(self, event: Optional[Any], project: GitProject) -> bool:
             account_name = event.github_login
             if not account_name:
                 raise KeyError(f"Failed to get account_name from {type(event)}")
-
-            if not self.is_approved(account_name):
-                logger.error(f"User {account_name} is not approved on whitelist!")
+            extra_condition = False
+            namespace = None
+            if isinstance(event, PullRequestEvent):
+                namespace = event.base_repo_namespace
+                extra_condition = self.is_approved(namespace)
```

Comment:
I don't understand why we check the `is_approved(namespace)` in case of `PullRequestEvent`, but not in `PullRequestCommentEvent` case. Can you comment (in code) it please?

Reply:
think about it :P

Reply:
okay, I thought about it and realized, I tried to combine whitelist+collaborator check here